### PR TITLE
tiup: add a notice to tiup dm import

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Make clear that this command is only used for upgrading DM v1.0 to later versions.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [x] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-component-dm-import.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 title: tiup dm import
 ---
```
Comment (ran-huang) [inline tiup/tiup-component-dm-import.md:4]: preview: https://pingcap-docs-website-scaffold-1hq.pages.dev/zh/tidb/stable/tiup-component-dm-import

File: tiup/tiup-component-dm-import.md [hunk 2]
```diff
@@ -2,7 +2,11 @@
...
 
-# tiup dm import
+# tiup dm import <span class="version-mark">仅适用于 DM v1.0</span>
```
Comment (ran-huang) [inline tiup/tiup-component-dm-import.md:5]:
```suggestion
# tiup dm import <span class="version-mark">仅适用于升级 DM v1.0</span>
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**